script/entry.js

```diff
@@ -8,6 +8,8 @@ const requireBabelConfig = require('../src/babel-config');
 
 requireBabelConfig(extensions);
 
+const { disableArchiveTreeCaching } = require('../src/app/content/utils/archiveTreeUtils');
```

Comment:
this will make pre-rendering very slow, i think we should only call it on the script you have that is dealing with the two archive versions